src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/karma.conf.js

```diff
@@ -26,6 +26,6 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: ['Chrome'],
-    singleRun: false
+    singleRun: true
```

Comment:
I'd think the config here should be the same as whatever is the default in Angular CLI, since this is what ends up in user projects. Is the original default `true`?

Reply:
The original default is false. I changed it because `singleRun: false` runs in an interactive mode which 1) I find super annoying and 2) Prevents any script from continuing onward. You're correct that we should try to keep it close to the original, so I'll change it back, but I'm sad about it.